bw2io/modified_database.py

```diff
@@ -7,27 +7,72 @@
 
 
 class ModifiedDatabase(object):
-    """Find relationships between foreground data ``data`` and background database named ``ref_database_name``.
+    # """Find relationships between foreground data ``data`` and background database named ``ref_database_name``.
```

Comment:
...same question here: [why are you keeping the old docstrings in and commenting them out?](https://github.com/brightway-lca/brightway2-io/pull/169/files#r1141269894)